lib/rules/links/linkchecker.js

```diff
@@ -14,8 +14,8 @@ const compound = {
 };
 
 const allowList = [
-    new RegExp('^https://www.w3.org/StyleSheets/TR/2016/'),
-    new RegExp('^https://www.w3.org/scripts/'),
+    new RegExp('^https://www.w3.org/StyleSheets/TR/'),
+    new RegExp('^https://www.w3.org/scripts/TR/'),
```

Comment:
We can allow any scripts under `/scripts/`. We do have control over that folder.
```suggestion
    new RegExp('^https://www.w3.org/scripts/'),
```